parler_rest/fields.py

```diff
@@ -3,6 +3,8 @@
 Custom serializer fields for nested translations.
 """
 from __future__ import unicode_literals
+import json
+from past.builtins import basestring
```

Comment:
Instead of including a new dependency, could you please use `django.utils.six.string_types` ?